<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCustomDialog</class>
 <widget class="QDialog" name="CCustomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout" columnstretch="2,1">
     <item row="0" column="0">
      <widget class="QLineEdit" name="lineEdit_1"/>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinBox_1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBox_2"/>
     </item>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit_3"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBox_3"/>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinBox_4"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
